ui/pages/confirmations/components/confirm/footer/footer.tsx

```diff
@@ -153,12 +157,16 @@ const Footer = () => {
     }
   }, [currentConfirmation, customNonceValue]);
 
+  const onFooterCancel = useCallback(() => {
+    onCancel({ location: MetaMetricsEventLocation.Confirmation });
+  }, [currentConfirmation]);
```

Comment:
Also here, I think we should include `onCancel` as a dependency.

Reply:
done